ajax/libs/blueprintjs-core/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "blueprintjs-core",
+  "filename": "blueprint.min.css",
+  "description": "A React UI toolkit for the web.",
+  "keywords": [
+    "palantir",
+    "blueprint",
+    "components",
+    "styles",
+    "theme",
+    "ui"
+  ],
+  "author": "Palantir Technologies",
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/palantir/blueprint"
+  },
+  "homepage": "http://blueprintjs.com/",
+  "npmName": "@blueprintjs/core",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/+(blueprint*|core.bundle*)"
+      ]
+    },
+    {
+      "basePath": "",
+      "files": [
+        "resources/icons/*.+(woff|eot|ttf)"
```

Comment:
This does not match anything?

Reply:
As I see it is exists in version below 3.x, so I think better to fetch this one